Sources/Extensions/Codable/Encodable+Extensions.swift

```diff
@@ -35,4 +35,20 @@ public extension Encodable {
 
         return json
     }
+
+    /// Method that converts encodable model to URLQueryItems array
+    /// - Returns: URLQueryItems array
+    func asUrlQueryItems() throws -> [URLQueryItem] {
+        return try toJSON().map {
+            if let value = $1 as? Encodable,
```

Comment:
toJSON creates dictionary with primitive types so there should'n be any custom Encodable types. isn't it?